desktop/src/main/java/bisq/desktop/main/overlays/windows/SignSpecificWitnessWindow.java

```diff
@@ -194,15 +161,24 @@ private void addSuccessContent() {
 
     @Override
     protected void addButtons() {
-        var buttonTuple = add2ButtonsAfterGroup(gridPane, ++rowIndex + 1,
+        var buttonTuple = add2ButtonsAfterGroup(gridPane, ++rowIndex + 2,
                 Res.get("popup.accountSigning.singleAccountSelect.headline"), Res.get("shared.cancel"));
 
         actionButton = buttonTuple.first;
         actionButton.setDisable(true);
-        actionButton.setOnAction(e -> selectAccountAgeWitness());
+        actionButton.setOnAction(e -> importAccountAgeWitness());
 
         closeButton = (AutoTooltipButton) buttonTuple.second;
         closeButton.setOnAction(e -> hide());
+    }
 
+    private ECKey checkedArbitratorKey() {
+        var arbitratorKey = arbitratorManager.getRegistrationKey(privateKey.getText());
```

Comment:
So that use old arbitrators, right? there are none registered anymore. Wouldn't it be better to use mediators or refund agents?

Reply:
It might be but that's a bigger change and I'm not convinced. I think that would add another complication to adding more mediators.